test/test-normalise-data.js

```diff
@@ -16,7 +16,12 @@ describe('normalise data', function() {
         client = await database_pool.connect();
         let publisher_feed;
         try {
+            /* Reset the database state in case of stale data from other tests */
+            await client.query('DELETE FROM raw_data');
```

Comment:
As a final note, are there docs about running the tests? I assume this is running against a test db but didn't find the docs from a quick scan (that was perhaps my own fault, however)

Reply:
I couldn't find it either, so I've added a note in the README.